packages/ember-router-helpers/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "ember-router-helpers",
+  "version": "0.4.0",
+  "description": "Template helpers utilizing the router service.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "homepage": "https://github.com/rwjblue/ember-router-helpers#readme",
+  "bugs": {
+    "url": "https://github.com/rwjblue/ember-router-helpers/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rwjblue/ember-router-helpers.git"
+  },
+  "license": "MIT",
+  "author": "Robert Jackson <me@rwjblue.com>",
+  "exports": {
+    ".": "./dist/index.js",
+    "./*": "./dist/*",
+    "./test-support": "./dist/test-support/index.js",
+    "./addon-main.js": "./addon-main.js"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "copyDoc": "cp ../../*.md .",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepare": "npm-run-all --aggregate-output --continue-on-error --parallel copyDoc build"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.6.0",
+    "ember-modifier": "^3.0.0"
```

Comment:
@ctjhoa it's not needed here, right?

Reply:
Yep you're right. Fixed https://github.com/rwjblue/ember-router-helpers/pull/293/commits/61d9d86d8d044edcbd076010a325780c69828087